src/software/ai/hl/stp/play/play_factory.cpp

```diff
@@ -9,5 +9,5 @@ std::unique_ptr<Play> createPlay(const TbotsProto::Play& play_proto,
                                  std::shared_ptr<const AiConfig> ai_config)
 {
     return GenericFactory<std::string, Play, AiConfig>::create(
-        TbotsProto::Play_Name(play_proto), ai_config);
+        TbotsProto::Play_PlayName_Name(play_proto.name()), ai_config);
```

Comment:
```suggestion
        TbotsProto::Play::PlayName_Name(play_proto.name()), ai_config);
```
Makes readability slightly better

Reply:
agreed